test/classes/initializers/phase1/badDependence.good

```diff
@@ -0,0 +1,2 @@
+badDependence.chpl:11: error: explicit initialization of field "f2" relies on later field
+badDependence.chpl:12: error: explicit initialization of field "f3" relies on later field
```

Comment:
Are we still trying for Phase 1 must be in field order? If so, wouldn't the error message be clearer if it said "Phase 1 initialization does not match field declaration order"?